cypress/e2e/eholdings/eholdings-titles.cy.js

```diff
@@ -21,11 +21,13 @@ describe('eHoldings titles management', () => {
   let userId;
 
   it('C16994 Add a title in a package to holdings (spitfire)', { tags: [testTypes.smoke, devTeams.spitfire, features.eHoldings] }, () => {
-    cy.createTempUser([permissions.uieHoldingsRecordsEdit.gui,
-      permissions.uieHoldingsPackageTitleSelectUnselect.gui]).then(userProperties => {
+    cy.createTempUser([
+      permissions.uieHoldingsRecordsEdit.gui,
+      permissions.uieHoldingsPackageTitleSelectUnselect.gui,
+      permissions.uieHoldingsTitlesPackagesCreateDelete.gui
+    ]).then(userProperties => {
       userId = userProperties.userId;
-      cy.login(userProperties.username, userProperties.password);
-      cy.visit(TopMenu.eholdingsPath);
+      cy.login(userProperties.username, userProperties.password, { path: TopMenu.eholdingsPath, waiter: () => eHoldingsTitlesSearch.waitLoading() });
```

Comment:
you can write just

`waiter: eHoldingsTitlesSearch.waitLoading`

there is no need arrow function